Obsidian.API/_Interfaces/IConfig.cs

```diff
@@ -58,4 +58,35 @@ public interface IConfig
     /// Paths of plugins that are loaded at the starttime.
     /// </summary>
     public string[] DownloadPlugins { get; set; }
+
+    /// <summary>
+    /// Enabled Remote Console operation.
+    /// </summary>
+    /// <remarks>See more at https://wiki.vg/RCON</remarks>
+    public bool EnableRcon { get; set; }
+
+    /// <summary>
+    /// Password to access the RCON.
+    /// </summary>
+    public string RconPassword { get; set; }
+
+    /// <summary>
+    /// Port on which RCON server listens.
+    /// </summary>
+    public ushort RconPort { get; set; }
+
+    /// <summary>
+    /// Whether the RCON commands should be sent to currently online Operators.
+    /// </summary>
+    public bool BroadcastRconToOps { get; set; }
+
+    /// <summary>
+    /// Key used in RCON encryption.
+    /// </summary>
+    public string? RconKey { get; set; }
+
+    /// <summary>
+    /// Whether the RCON encryption can happen with a Diffie-Hellman Key Exchange or only PSK.
+    /// </summary>
+    public bool AllowDiffieHellman { get; set; }
```

Comment:
Considering encryption is a fragile subject, this should be an enum to allow for future changes.

Reply:
By enum do you mean only one encryption method or multiple possibilities at once